apps/web/src/pages/Home/HomePage/components/Profile/Profile.tsx

```diff
@@ -77,17 +77,18 @@ const Profile = () => {
             data-h2-gap="base(x1)"
             data-h2-justify-content="base(center) p-tablet(flex-start)"
           >
-            <CallToActionLink
+            <Link
               color="quinary"
+              mode="cta"
               href={paths.myProfile()}
-              Icon={ProfileIcon}
+              icon={UserGroupIcon}
```

Comment:
Intentional change?
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/333f67e1-3cdb-43c0-9af3-b2c058f88cfe)

Reply:
Not intentional 😬 Reverted in [fix incorrect profile icon](https://github.com/GCTC-NTGC/gc-digital-talent/commit/69bb1914b451643370da3b12b817252a5305aba3)